packages/mesos-metrics-module/build

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+mkdir -p build
+pushd build
+
+# configure module build with cmake
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DTESTS_ENABLED=False \
+  -DCMAKE_INSTALL_PREFIX=$PKG_PATH \
+  -Dmesos_INCLUDE_DIR=/opt/mesosphere/include \
+  -Ddefault_BIN_DIR=/opt/mesosphere/bin \
+  -Ddefault_LIB_DIR=/opt/mesosphere/lib \
+  -DUSE_LOCAL_PROTOBUF=False \
+  -Dprotobuf_INCLUDE_DIR=/opt/mesosphere/lib/mesos/3rdparty/include \
+  -DUSE_LOCAL_GLOG=False \
+  -Dglog_INCLUDE_DIR=/opt/mesosphere/lib/mesos/3rdparty/include \
+  -DUSE_LOCAL_AVRO=False \
+  /pkg/src/mesos-metrics-module/mesos_module/
+
+make -j8
+
+# install lib to $PKG_PATH/...
+make install
+
+popd
```

Comment:
there isn't any value in doing the popd here (all we do is copy one file after this with absolute paths everywhere....)